README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
 
+    mscs-server-jar=BungeeCord.jar
+    mscs-server-url=http://ci.md-5.net/job/BungeeCord/lastSuccessfulBuild/artifact/bootstrap/target/BungeeCord.jar
 
-## Usage
+##### Additional documentation
+
+More examples and documentation on server customization can be found on the [wiki](https://github.com/sandain/MinecraftServerControlScript/wiki/Server-Customization-Examples) page.
+
+### The msctl file
+The `msctl` file can be found at `/usr/local/bin`.
+
+The `msctl` file is an additional file for adjusting the properties of your server that are not found in `mscs.properties`. Starting at line 236 and ending at line 385 is where the settings are in this file.
+
+You can set the default settings of **mscs.properties** in this file, as well as some other useful settings:
+* Line 305 is the beginning of the **server instance configuration** settings.
+* Line 321 is the beginning of the **backup configuration** settings.
+* Line 334 is the beginning of the **server log configuration** settings.
+* Line 341 is the beginning of the **listing option** settings.
+* Line 348 is the beginning of the **mirror image** settings.
+* Line 370 is the beginning of the **minecraft overviewer** settings
+
+## Automated backups and restarts
+
+### Scheduling backups
+MSCS uses [**cron**](https://en.wikipedia.org/wiki/Cron), a scheduler software that can run programs on a set interval of time. Below is an example of one way how you could setup backups via cron to backup a world every 2 hours:
+
+Type the following (in any directory): 
+
+  ````
+  export EDITOR=vim
+  crontab -e
+  ````
+  Page down until you get to an empty line. Then paste the following:
+  
+  ````
+  0 */2 * * *  PATH-TO-MSCS backup myWorld
+  ````
+  * `0 */2 * * *` is the time interval to backup. This particular expression means backup every 2 hours. You can change this to 3, 4, 5 or whatever amount of hours to backup X amount of hours. You can also backup according to days, minutes, seconds, the time of the day, and more. See [the wiki page](https://github.com/sandain/MinecraftServerControlScript/wiki/Backup-and-Restore) for more information.
+  * `PATH-TO-MSCS` is the path of where the `mscs` file is (which will be inside the `MinecraftServerControlScript` folder that you downloaded)
+  * `myWorld` is the name of the world you wish to backup. Omitting this will backup all worlds.
+
+  For instance, if the `mscs` script is located in `/home/MinecraftServerControlScript/mscs`, and I want to backup the world "vanillaMC" every 2 hours, it would look like this:
+  
+  ````
+  0 */2 * * *  /home/MinecraftServerControlScript/mscs backup vanillaMC
+  ````
+  
+  Finally, press escape, then type
+  `:wq`
+  to save and quit.
+  
+  The backups will be saved in `/opt/mscs/backups`. 
+  
+### Removing backups after X days
+You can specify how long to keep backups by changing the `BACKUP_DURATION` in the `/usr/local/bin/msctl` file, on line 331.
```

Comment:
Repeating, `/usr/local/bin/msctl` should not be edited, point to the locations of the loadable config locations.